downstream/assemblies/platform/assembly-set-up-automation-mesh.adoc

```diff
@@ -0,0 +1,25 @@
+
+ifdef::context[:parent-context: {context}]
+
+
+[id="set-up-automation-mesh"]
+= Setting up {AutomationMesh}
+
+
+:context: set-up-automation-mesh
+
+
+[role="_abstract"]
+The following sections provide information on how to set up automation mesh for your Ansible environment by configuring the {PlatformNameShort} installer, as well as an option to import your own Certificate Authority (CA) certificate.
```

Comment:
Change "automation mesh" to the product variable.

Reply:
This makes sense. Thanks for breaking it down for me